doc/components/EventComponent.js

```diff
@@ -24,7 +24,7 @@ EventComponent.Prototype = function() {
           .on('click', this.onClick)
           .append($$('span').addClass('se-name').append(node.name)),
         $$('div').addClass('se-source').append(
-          $$('strong').append(this.i18n.t('event')),
+          $$('strong').append(this.getLabel('event')),
```

Comment:
Actually see doc/i18n/en.js for all the translations. Just move them into DocumentationReaderPackage.js. And when done and things look good you can delete this doc/i18n/en.js

Same is true for substance/i18n/en.js + de.js. Just wipe it once all translations are done via config.addLabel.